capitoli/Settimo_Passo_Funzioni.rst

```diff
@@ -74,9 +72,9 @@ In pratica dobbiamo scrivere un breve programma che esegua le operazioni che abb
 
 **Tutto questo andrà scritto su una sola riga, che sarà la prima linea della funzione.**
 
-|
-| :boltblue:`DEFINIZIONE DI FUNZIONE`
-|
+-----------------------------------
+:boltblue:`DEFINIZIONE DI FUNZIONE`
+-----------------------------------
```

Comment:
Se vogliamo usarlo come titolo allora può andar bene ma metti solo la prima lettera in maiuscolo.